clients/drcachesim/tests/raw2trace_unit_tests.cpp

```diff
@@ -1508,6 +1508,82 @@ test_rseq_rollback_with_signal(void *drcontext)
         check_entry(entries, idx, TRACE_TYPE_FOOTER, -1));
 }
 
+/* Tests rseq rollback with taken branch right before signal. */
+bool
+test_rseq_rollback_with_signal_after_branch(void *drcontext)
+{
+    std::cerr << "\n===============\nTesting legacy rseq rollback with taken branch "
```

Comment:
Delete "legacy" as this does have the rseq entry marker.

Reply:
fixed.